docs/admin/node.md

```diff
@@ -165,6 +165,11 @@ completely unhealthy (i.e. there are no healthy nodes in the cluster). In such
 case, the node controller assumes that there's some problem with master
 connectivity and stops all evictions until some connectivity is restored.
 
+In Kubernetes 1.6, we introduced the logic for the node controller to handle
+`taints` and `tolerance`. Basically, `taints` go on nodes while `tolerations`
```

Comment:
Consider rephrasing this sentence.

e.g. (Taints are applied to nodes and tolerations are applied to pods.)